index.js

```diff
@@ -152,7 +164,7 @@ function fastifyStatic (fastify, opts, next) {
   if (opts.serve !== false) {
     if (opts.wildcard === undefined || opts.wildcard === true) {
       fastify.get(prefix + '*', routeOpts, function (req, reply) {
-        pumpSendToReply(req, reply, '/' + req.params['*'])
+        pumpSendToReply(req, reply, '/' + req.params['*'], sendOptions.root)
```

Comment:
why this change?

Reply:
I think this was an early change I made that can be reverted. No valid reason I can think of to change it right now.